docassemble/EFSPIntegration/efm_client.py

```diff
@@ -104,7 +109,12 @@ def RegisterUser(self, person_to_reg, password:str, registration_type:str='INDIV
             'phoneNumber': person_to_reg.sms_number()[-10:]
         }
 
-        resp = self.proxy_client.put(self.base_url + 'adminusers/users', data=json.dumps(reg_obj))
+        send = lambda: self.proxy_client.put(self.base_url + 'adminusers/users', data=json.dumps(reg_obj))
+        resp = send()
```

Comment:
I can change on my branch to match. Just curious--why the change to a lambda instead of directly invoking?

Reply:
Just so you don't have to type out the exact same call to the server twice if it fails the first time.